pom.xml

```diff
@@ -103,6 +103,7 @@
         <commons-configuration.version>1.10</commons-configuration.version>
         <javax-mail.version>1.4.4</javax-mail.version>
         <powermock.version>1.6.6</powermock.version>
+        <nitf-imaging.version>0.5</nitf-imaging.version>
```

Comment:
@glenhein you'll need to update this to 0.6 now that this PR is merged: https://github.com/codice/alliance/pull/221